cypress/fixtures/mockTypes.ts

```diff
@@ -6,7 +6,7 @@ import {
 } from '@graasp/sdk';
 
 export type MockItemTag = Omit<ItemTag, 'item'>;
-export type MockItem = Omit<PackedItem, 'permission' | 'hidden' | 'public'> & {
```

Comment:
Why was this changed ?

Reply:
I think we should remove the omit anyway, but I will set back the permission for now.

Reply:
I was not agains the change, I just could not see why it was done. If it is not necessary to omit permission, then you can keep the change no problem.